parse_redcap_errors_err.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+import sys
+import re
+from glob import glob
+from tqdm import tqdm
+
+# Example Usage:
+# python parse_redcap_errors_err.py /path/to/redcap-import-<JOBID>-*.err
+# ./parse_redcap_errors_err.py /path/to/redcap-import-<JOBID>-*.err
+
+def parse_file(file_path) -> tuple:
+    """
+    Parses a file for errors and returns the file path and the number of errors found.
+    Expects redcap-import-*-*.err files.
+
+    Args:
+        file_path (str): The path to the file to parse.
+
+    Returns:
+        tuple: A tuple containing the file path and the number of errors found.
+    """
+    try:
+        with open(file_path) as file:
+            content = file.read().split('\n')
+            error_count = 0
+
+        idx = 0
+        while idx < len(content):
+            line = content[idx]
+            # If the line contains 'Traceback (most recent call last):', then there was an error
+            if 'Traceback (most recent call last):' in line:
+                error_count += 1
```

Comment:
`error_count` code should be remove. This is unnecessary keeping in mind the simplicity of this program.